src/python/bcc/__init__.py

```diff
@@ -1093,7 +1093,12 @@ def cleanup(self):
             # non-string keys here include the perf_events reader
             if isinstance(k, str):
```

Comment:
Oops, dcsnoop should probably be converted as a performance optimization. Returning 1 there causes more work on the kernel side, and I can tell from the python code that you don't need the perf event in userspace. The others are fine since they are just returning 1 from an internal static inline helper.